bitsail-connectors/bitsail-connectors-legacy/bitsail-connector-jdbc/src/main/java/com/bytedance/bitsail/connector/legacy/jdbc/converter/JdbcValueConverter.java

```diff
@@ -369,10 +364,10 @@ protected Double convertDoubleValue(Object value, String columnName, String colu
       return ((BigDecimal) value).doubleValue();
     }
     throw new IllegalArgumentException(String
-        .format("Unexpected value for JDBC type: %s and column %s", columnTypeName, columnName));
```

Comment:
This debug message is quite useful to local the column that has an issue. Is there any reason we wanna make this change?

Reply:
The reason I made this change was because I don't see a logic usage besides logging for `columnName` and `columnTypeName`. However it makes sense if we want those specific logs to easily locate issue. Will add `columnName` and `columnTypeName` back.
One follow-up question for this log: Original logging doesn't printout `value`. Is there any reason (such as privacy concern) for doing so? If no then I can printout `value` to help debugging.

Reply:
We don't have a standard whether to log `value` or not, some connectors does.